### PR TITLE
[WIP] [DOC] describe content of cfg and of checkCFG 

### PR DESCRIPTION
**Pull request by Remi-Gau**
## TODO
- [ ] describe content and use of cfg in rst doc
- [x] describe what checkCFG does in the help section

### CODE REVIEW

**Comment by Remi-Gau** [comment]
@CerenB @marcobarilari 
I still need to update the actual sphinx documentation related to cfg. But you can already start looking into the new and "very small" help section of checkCFG.

**Review by marcobarilari — APPROVED** [review]
LGTM :)

**Comment by Remi-Gau** [comment]
@marcobarilari @CerenB 

I updated the `index.rst` and created a `set_up.rst` and tried to move some of the content that was in the markdown in there.

Maybe have a look and let me know what you think.

File: docs/source/gui.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  
+List of functions in the ``gui`` folder: those are mostly to handle the "graphic interface"
+that can be used to deal with ``userInputs()``.
+
+----
+
+.. automodule:: src.gui 
+
+.. autofunction:: askForGroupAndOrSession
+.. autofunction:: askUserCli
```
Comment (marcobarilari) [inline docs/source/gui.rst:12]: this one for example is not for the GUI, it could be confusing ti have it there

**Comment by marcobarilari** [comment]
overall LGTM

File: docs/source/gui.rst (hunk 1 above)
Comment (Remi-Gau) [reply docs/source/gui.rst:12]: Good point.

**Comment by Remi-Gau** [comment]
> overall LGTM

OK will merge then. I am fighting hard my inner perfectionist to keep improving this.

At least all of that gives us a better base to start from in general.

File: docs/source/gui.rst (hunk 1 above)
Comment (Remi-Gau) [reply docs/source/gui.rst]: This should be fixed now.

**Pull request merged**